src/main/java/it/gov/pagopa/ranking/dto/initiative/InitiativeBuildDTO.java

```diff
@@ -29,5 +29,7 @@ public class InitiativeBuildDTO {
     private InitiativeBeneficiaryRuleDTO beneficiaryRule;
     @JsonProperty("initiativeRewardType")
     private String initiativeRewardType;
+    @JsonProperty("organizationName")
+    private String organizationName;
```

Comment:
let's keep fields ordered in order to find them more easily, put this after organizationId field